search/engine.go

```diff
@@ -473,9 +479,13 @@ func (e *ESEngine) DoSearch(ctx context.Context, query Query, sortBy string, fro
 	results := make([]*elastic.SearchResult, 0)
 	for _, lang := range query.LanguageOrder {
 		if r, ok := resultsByLang[lang]; ok {
-			results = r
-			currentLang = lang
-			break
+			if hasSpanish {
+				results = append(results, resultsByLang[lang]...)
```

Comment:
You don't touch any scoring, right?
Which is ok I think.

Reply:
Where are the different results being ordered by score?
Where that happens?

Reply:
Scores was not touched.
Ordering by score is inside joinResponses func. (using compareHits).